README.md

```diff
@@ -27,7 +27,7 @@ First, add HTTPoison to your `mix.exs` dependencies:
 
 ```elixir
 def deps do
-  [{:httpoison, "~> 0.10.0"}]
+  [{:httpoison, "~> 0.11.0"}]
```

Comment:
I would just put [{:httpoison, "~> x.x.x"}]  and add a sentence to look to the most recent stable version listed in releases.